assets/js/pages/HostDetailsPage/ProviderDetails.test.jsx

```diff
@@ -43,4 +45,80 @@ describe('Provider Details', () => {
       'Provider not recognized'
     );
   });
+
+  describe('AWS', () => {
+    const metadataLabels = [
+      {
+        key: 'account_id',
+        label: 'Account ID',
+      },
+      {
+        key: 'ami_id',
+        label: 'AMI ID',
+      },
+      // availability_zone is not displayed on its own but as part of the region
+      // {
+      //   key: 'availability_zone',
+      //   label: null,
+      // },
+      {
+        key: 'data_disk_number',
+        label: 'Data disk number',
+      },
+      {
+        key: 'instance_id',
+        label: 'Instance ID',
+      },
+      {
+        key: 'instance_type',
+        label: 'Instance type',
+      },
+      {
+        key: 'region',
+        label: 'Region',
+      },
+      {
+        key: 'vpc_id',
+        label: 'VPC ID',
+      },
+    ];
+
+    it('should render with AWS metadata', () => {
+      const awsMetadata = {
```

Comment:
nitpick: ideally we would have this in a factory. But well, I'm not going to complain if you don't. You did more than others adding the test at least hehe